litebird_sim/mapmaking/binner.py

```diff
@@ -259,7 +286,9 @@ def make_binned_map(
             to use for the output map
         components (list[str]): list of components to include in the map-making.
             The default is just to use the field ``tod`` of each
-            :class:`.Observation` object.
+            :class:`.Observation` object
+        detector_split (str): select the detector split to use in the map-making
+        time_split (str): select the time split to use in the map-making.
```

Comment:
Once we have settled on a definitive list of splits we should list the accepted values in the docstrings, so that users do not need to resort to the manual to find out what to put in these arguments.

(Of course, this does not apply if we decide to use an enumeration here, as they are self-documenting.)

Reply:
Hello @ziotom78, in this direction I implemented a function that checks the validity of the requested splits.
For example, it checks that a 1-year split fits in the requested observation duration, and so on.

I am still developing stuff of course, but the eventual documentation of this function should cover the valid splits etc, as you were suggesting.